### PR TITLE
remove tutorial rule references

### PR DESCRIPTION
**Pull request by Bee-lee**
# Description
tutorial_rule has long been deprecated. Since we have the `update_rules_content.sh` script and the test rule, there's no need to keep the tutorial rule referenced. 

this change won't affect smart-proxy, as it was always removing rules with incomplete content. 

Fixes CCXDEV-11618

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- Refactor (refactoring code, removing useless files)

## Testing steps
`make test`

## Checklist
* [x] `make before_commit` passes
* [x] updated documentation wherever necessary
* [x] added or modified tests if necessary
* [x] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 set -exv
 
 # retrieve the latest tag set in repository
-version=$(git describe --tags --abbrev=0)
+version=$(git describe --always --tags --abbrev=0)
```
Comment (tisnik) [inline build.sh:19]: what it does pls?

**Review by tisnik — APPROVED** [review]
LGTM, just a question

File: build.sh (hunk 1 above)
Comment (Bee-lee) [reply build.sh:19]: resolved in slack

Comment (epapbak) [reply build.sh:19]: can you point me to the discussion please?

**Review by epapbak — APPROVED** [review]

**Review by juandspy — APPROVED** [review]

**Pull request merged**